### PR TITLE
added task solution

### PR DESCRIPTION
**Pull request by AnyaRatusnyak**

### CODE REVIEW

File: src/Ticket.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    private String id;
+    private String concertHall;
+    private int eventCode;
+    private long time;
+    private boolean isPromo;
+    private String stadiumSector;
+    private BigDecimal weight;
+    private double price;
+
+    private static final BigDecimal MAX_WEIGHT = new BigDecimal("1.000");
```
Comment (Alexmansar) [inline src/Ticket.java]: in general this also could be just double))

File: src/Ticket.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+            int eventCode
+    ) {
+        if ( concertHall.length() > 10) {
+            throw new IllegalArgumentException("Concert hall name must contain max 10 characters.");
+        }
+        this.concertHall = concertHall;
+        if (eventCode < 100 || eventCode > 999) {
+            throw new IllegalArgumentException("Event code must be a 3-digit number.");
+        }
+        this.eventCode = eventCode;
```
Comment (Alexmansar) [inline src/Ticket.java]: we have a repeating code. These can be separate methods)

File: src/Ticket.java [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        this.time = System.currentTimeMillis();
```
Comment (Alexmansar) [inline src/Ticket.java]: pls, look at the line 36))

File: src/Ticket.java [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    }
+
+    public Ticket() {
+    }
```
Comment (Alexmansar) [inline src/Ticket.java]: in rhis constructor we also can add
this.time = System.currentTimeMillis()/1000;

**Review by Alexmansar — CHANGES_REQUESTED** [review]
Overall, good job) Just a few comments

File: src/Ticket.java (hunk 4 above)
Comment (AnyaRatusnyak) [reply src/Ticket.java]: Thank you ! I fixed my solution)

**Review by Alexmansar — APPROVED** [review]
@AnyaRatusnyak Good job)) could be merged))

**Pull request merged**